holoviews/tests/plotting/bokeh/test_barplot.py

```diff
@@ -276,3 +277,75 @@ def test_bars_color_index_color_clash(self):
             "mapping for 'color' option and declare a color_index; ignoring the color_index.\n"
         )
         self.assertEqual(log_msg, warning)
+
+    def test_bars_continuous_data_list_same_interval(self):
+        bars = Bars(([0, 1, 2], [10, 20, 30]))
+        plot = bokeh_renderer.get_plot(bars)
+        data = plot.handles["source"].data
+        np.testing.assert_almost_equal(data["width"], 0.2)
```

Comment:
```suggestion
        assert data["width"] == 0.2
```

Unless I'm missing something